app/helpers/layout_helper.rb

```diff
@@ -21,7 +21,7 @@ def stylesheet(*args)
   end
 
   def javascript(*args)
-    content_for(:javascripts) { javascript_include_tag(*args) }
+    content_for(:javascripts) { javascript_include_tag(*args.push("data-turbolinks-track" => true)) }
   end
```

Comment:
@isratrade I think we probably need the track also on application.js and css in the default layout, as I'm not sure if it would survive a foreman upgrade session?